src/openMVG/sfm/sfm_data_filters_frustum.cpp

```diff
@@ -23,8 +23,10 @@ using namespace openMVG::geometry::halfPlane;
 
 // Constructor
 Frustum_Filter::Frustum_Filter(const SfM_Data & sfm_data,
-  const double zNear, const double zFar)
+  const double zNear, const double zFar, const NearFarPlanesT & z_near_z_far)
 {
+  z_near_z_far_perView = z_near_z_far;
```

Comment:
Forgot to mention, since we are in the constructor you can use the initializer list

```
Frustum_Filter::Frustum_Filter
(
  const SfM_Data & sfm_data,
  const double zNear,
  const double zFar,
  const NearFarPlanesT & z_near_z_far
)
:  z_near_z_far_perView(z_near_z_far)
{
  ...
}
```